src/app/admin/core/core.module.ts

```diff
@@ -1,13 +1,13 @@
 import { NgModule, ErrorHandler } from '@angular/core';
-import { ModuleWithProviders } from '@angular/compiler/src/core';
+import { ModuleWithProviders } from '@angular/core';
 
 import { PrivilegeService } from './privilege.service';
 
 @NgModule({
   exports: []
 })
 export class CoreModule {
-  static forRoot(): ModuleWithProviders {
+  static forRoot(): ModuleWithProviders<any> {
```

Comment:
Is there a type we can use here or is this something we have define?

Reply:
Quick Answer: Yes I can type this and all other instances of modules.

Story: Type enforcement on the module exports was introduced in the 9 --> 10 upgrade. The problem I had then was that the type had to be 'CoreModule' (in this case) but the way we exported modules was just as a class without the @NgModule decorator. I did not manually update these module files to the new format so I just left it as is with type any. While upgrading between 10 --> 13, the ng update handled this decorator formatting for me so now I can go through and properly type all these modules.